pkg/agent/controller/networkpolicy/fqdn_test.go

```diff
@@ -46,6 +52,7 @@ func newMockFQDNController(t *testing.T, controller *gomock.Controller, dnsServe
 		true,
 		false,
 		config.DefaultHostGatewayOFPort,
+		clockToInject.(clock.WithTicker),
```

Comment:
change type of `clockToInject` parameter to `clock.WithTicker`, and remove this type assertion